atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -152,6 +161,11 @@ public T getFirst(@Nullable T defaultElement) {
         return BatchingVisitables.getFirstPage(delegate(), numToVisitArg);
     }
 
+    /**
+     * Gets the last element in the visitable.
+     *
+     * @return the last element or null if the visitable is empty
+     */
     @Nullable
     public T getLast() {
```

Comment:
Should add javadoc to getLast(defaultElement) and link to it from here or vice versa.